themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Review Stacks: Collaborate in the Cloud"
+date: 2023-06-14
+meta_desc: Review Stacks give you an automated cloud environment for every pull request so your team can ship faster, safer, and more collaboratively.
+meta_image: meta.png
+authors:
+    - evan-boyle
+    - pat-gavlin
+tags:
+    - cloud engineering
+    - pulumi deployments
+    - automation api
+    - deployments
+    - review stacks
+---
+
+At Pulumi, our engineering team writes *a ton* of code. We have hundreds of repositories with programs managing thousands of stacks and tens of thousands of cloud resources. These tools and services power production workloads, internal tools, business intelligence and process, and more. When you’re an engineer looking to work in one of these repositories, making a code change is usually the easy part. The hard part is figuring out how to get a tight dev loop going. Are there instructions in the README for running locally? A docker compose file? Can I figure out what config and upstream stack references are required to stand up a fresh dev stack? We make it incredibly easy to build in the cloud and write prolific amounts of code, and with that comes a need to standardize our development process.
```

Comment:
```suggestion
At Pulumi, our engineering team writes *a ton* of code. We have hundreds of repositories with programs managing thousands of stacks and tens of thousands of cloud resources. These tools and services power production workloads, internal tools, business intelligence and process, and more. And of course we use Pulumi to ship all of it. When you’re an engineer looking to work in one of these repositories, making a code change is usually the easy part. The hard part is figuring out how to get a tight dev loop going. Are there instructions in the README for running locally? A docker compose file? Can I figure out what config and upstream stack references are required to stand up a fresh dev stack? We make it incredibly easy to build in the cloud and write prolific amounts of code, and with that comes a need to standardize our development process.
```

Reply:
Possibly too lengthy, might be better to come up with a 2 sentence opening, and then make this the first real paragraph. For example:

> Pulumi has all the building blocks to productively deploy and manage cloud infrastructure using infrastructure as code. But as more engineers on the team need to collaborate, and the scope of infrastructure changes grows, it becomes necessary for engineers to test and review their changes before deploying them. This can be challenging---until now. Enter Review Stacks: a new capability that automatically provisions temporary infrastructure as code environments scoped to a Git pull request making collaboration fast and seamless.

Reply:
Great feedback, tried to incorporate the spirit of this into a new rev. Let me know what you think.